jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -120,7 +187,7 @@ The number of containers is specified by the [`parallelism` key](/docs/configura
 The current container index is automatically picked up from the `$CIRCLE_NODE_INDEX` environment variable, but can be manually set by using the `--index` flag.
 
 ```shell
-circleci tests split --index=0 test_filenames.txt
+circleci tests run --index=0 test_filenames.txt
```

Comment:
i think you'd need to rework this to be:

cat test_filenames.txt | circleci tests run --command=">index0.txt xargs echo" --index=0 --split-by=name

?